packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -7,6 +7,7 @@ import { WalletText } from '../WalletText';
 import './ModalStepWrapper.scss';
 
 type TModalStepWrapperProps = {
+    disableScroll?: boolean;
```

Comment:
This prop is designed for screens where scrolling isn't necessary, as it can cause odd interface behavior. For example, `PersonalDetails`.